packages/portfolio-contract/src/portfolio.exo.ts

```diff
@@ -4,7 +4,8 @@ import type { Zone } from '@agoric/zone';
 import { M } from '@endo/patterns';
 import { VowShape } from '@agoric/vow';
 import { atob, decodeBase64 } from '@endo/base64';
-import { decodeAbiParameters } from 'viem';
+// @ts-ignore vendored code
+import { decodeAbiParameters } from './vendor/viem/viem-abi.bundle.js';
```

Comment:
crud! this looked like it was working...

but we also import viem stuff from the orchestration package.

maybe I/we can move the vendor stuff down there?